test-cases/performance/perf-regression-latency-650gb-with-nemesis.yaml

```diff
@@ -33,9 +33,10 @@ print_kernel_callstack: true
 store_perf_results: true
 email_recipients: ["scylla-perf-results@scylladb.com"]
 use_prepared_loaders: true
-use_hdrhistogram: true
+use_hdr_cs_histogram: true
```

Comment:
It is old syntax . Now it should be  `use_hdrhistogram`. Please, reverse

Reply:
fixed